packages/slice-machine/src/modules/availableCustomTypes/index.ts

```diff
@@ -22,7 +22,7 @@ import {
   pushCustomTypeCreator,
   saveCustomTypeCreator,
 } from "../selectedCustomType/actions";
-import { saveCustomTypeSaga } from "../selectedCustomType/sagas";
+import { getCustomTypeStatus } from "server/src/api/custom-types/getCustomTypeStatus";
```

Comment:
this function should go into lib or src, so we don't risk passing server code to the front-end :) (we did that once by before :/)